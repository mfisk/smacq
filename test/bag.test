#!/bin/sh

${BINDIR}smacqq -g -m <<EOF 
print srcip, count() from (pcapfile($TEST/0325@1112-snort.pcap) + pcapfile($TEST/0325@1112-snort.pcap)) group by srcip
EOF
