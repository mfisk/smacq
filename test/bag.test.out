192.168.0.54	214
192.168.0.200	2
